### ISSUE TITLE
Option for the magnification of images 

### ISSUE THREAD

**Issue by matteosecli** [brucemiller/LaTeXML#942]
Images produces by LaTeXML are fine if you leave the CSS untouched. However, if you want to - say - increase the body text size at 150%, then the images look way too small. My current "dirty hack", since I define the width of all my images wrt `\textwidth`, is to put a `\addtolength{\textwidth}{0.5\textwidth}` after `\begin{document}` when I'm converting with LaTeXML; however, it's definitely an unorthodox solution which could lead to unexpected results sooner or later.

It would be nice, instead, to have an option which specifies the magnification factor for the images - much like you do with `--mathimagemagnification` for the math. The same goes for the picture environments.

**Comment by dginev** [comment]
We always have a fun time discussing new options for latexml :> Haven't added any in a while, maybe it's time for me to lower my guard on the subject? cc @brucemiller 

Personally, I am wondering if we can generalize option classes we already have, since `mathimagemagnification` has always read awkwardly to me. 22 characters is a mouthful. Just to get the ball rolling, could we upgrade towards a `--magnification=[mathimage:x;image:y]` or similar?

Another point is that LaTeX ought to give you all the power you need to specify relative sizes, and the latexml option set is not meant to reimplement the typesetting system, even in the cases where the analogue between the printed page and HTML starts feeling odd. If you start modifying the in-built CSS, it is pretty much your responsibility to make all changes needed to get a coherent appearance - something I have done for latexml-based projects in the past. But I agree it can be a pain, I'm still hunting for optimizations for the moderncv binding demo #924 which has somewhat similar kinds of concerns.

**Comment by matteosecli** [comment]
@dginev Thanks for the clarifications! 😃
I'm definitely not a CSS expert, but I stumbled upon the problem while using [TeX.js](https://github.com/davidar/TeX.js) (which does not require a great knowledge of such things). Things like TeX.js could be a great user-friendly tool to beautify LaTeXML output for casual users without having to actually look into LaTeXML; so I think that having a little bit more flexibility in this sense could be an advantage even for people who don't know how to modify a CSS (and LaTeXML itself), but still can easily use "modified" stiles.

**Comment by dginev** [comment]
Luckily the TeX.js founder is a latexml fan, and we collaborated a little while davidar was developing it. The best way to restyle the latexml output remains an open question, and I feel I should tie in a different conversation that is also pondering on the subject for the arxiv-vanity context:

https://github.com/arxiv-vanity/engrafo/issues/255

My take on this for now is that the obvious first step is to offer a detailed documentation of the HTML5 dialect, which should make writing new CSS at least partially easier (see #896), but that's just a part of the answer.

**Comment by dginev** [comment]
No time for option changes in 0.8.4, will bump to a further milestone until we get new option sets back on our minds.

**Comment by bfirsh** [comment]
Image sizing is a big issue that needs tackling in Engrafo. We haven't thought much about it yet, but we will at some point. Some background: https://github.com/arxiv-vanity/engrafo/issues/530

Some issues along the same lines, where LaTeXML is not quite doing what we want with the CSS: https://github.com/arxiv-vanity/engrafo/issues/250 https://github.com/arxiv-vanity/engrafo/issues/503

**Comment by dginev** [comment]
@brucemiller is this issue resolved through the new options from #1697 , in particular `magnify` ?

If so, maybe we can close. 

@matteosecli If you still have a test you'd like to check, you can try different values to see if this covers the basic need. The default starting point is:
```
$ latexmlc test.tex --dest=test.html --preload=[magnify=1,upscale=1,zoomout=1]latexml.sty
```

**Comment by brucemiller** [comment]
I'd like to hope so :>  there shouldn't be too many ways to achieve (roughly) the same end.
